index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
```

Comment:
The login form does not look good on a big screen. It occupies the whole viewport.